packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts

```diff
@@ -1,18 +1,31 @@
 import * as React from 'react';
+import { SliderBase } from './Slider.base';
+import { IStyle, ITheme } from '../../Styling';
+import { IStyleFunction } from '../../Utilities';
 
 export interface ISlider {
   value: number | undefined;
 
   focus: () => void;
 }
 
-export interface ISliderProps {
+export interface ISliderProps extends React.Props<SliderBase> {
   /**
    * Optional callback to access the ISlider interface. Use this instead of ref for accessing
    * the public methods and properties of the component.
    */
   componentRef?: (component: ISlider | null) => void;
 
+  /**
+   * Call to provide customized styling that will layer on top of the variant rules.
+   */
+  getStyles?: IStyleFunction<ISliderStyleProps, ISliderStyles>;
+
+  /**
+   * Theme provided by High-Order Component.
+   */
+  theme?: ITheme;
+
```

Comment:
Add `className?: string;` as well so consumers can add a class to the root component.

Reply:
`className?` is already available its just lower in the field set

https://github.com/KevinTCoughlin/office-ui-fabric-react/blob/5c9d141ca19e23a2d49dcfb469c09d602ff6ea88/packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts#L97